pkg/core/native/native_neo.go

```diff
@@ -249,17 +256,24 @@ func (n *NEO) OnPersist(ic *interop.Context) error {
 
 // PostPersist implements Contract interface.
 func (n *NEO) PostPersist(ic *interop.Context) error {
-	gas, err := n.GetGASPerBlock(ic, ic.Block.Index)
-	if err != nil {
-		return err
-	}
+	gas := n.GetGASPerBlock(ic.DAO, ic.Block.Index)
 	pubs := n.GetCommitteeMembers()
 	index := int(ic.Block.Index) % len(ic.Chain.GetConfig().StandbyCommittee)
 	gas.Mul(gas, big.NewInt(committeeRewardRatio))
 	n.GAS.mint(ic, pubs[index].GetScriptHash(), gas.Div(gas, big.NewInt(100)))
+	n.OnPersistEnd(ic.DAO)
```

Comment:
Both Policy and Oracle contracts have `OnPersistEnd` method too, but this method is called from `(bc *Blockchain) storeBlock` (see (link)[https://github.com/nspcc-dev/neo-go/blob/master/pkg/core/blockchain.go#L653]). Don't we want to unify `OnPersistEnd` calls for all native contracts?

Reply:
We do, but it's not that simple unfortunately because of neo-project/neo#1913. In general, explicit `OnPersistEnd` in `storeBlock` is a kludge, so if we can avoid it here, better avoid it.